src/main/java/com/cburch/logisim/circuit/appear/DefaultAppearance.java

```diff
@@ -31,7 +32,25 @@ public static List<CanvasObject> build(
   }
 
   static void sortPinList(List<Instance> pins, Direction facing) {
+    // Remove any clk pin and then add it to the end after sorting
+    Instance clk = null;
+    for (var it = pins.iterator(); it.hasNext(); ) {
+      var pin = it.next();
+      if (!pin.getAttributeSet().containsAttribute(StdAttr.LABEL)) {
+        continue;
+      }
+      var label = pin.getAttributeValue(StdAttr.LABEL);
+      if (label.equals("clk")) {
+        clk = pin;
```

Comment:
I would at least also allow for CLK, so do a down case of the label. Maybe also CLOCK and clock should be candidates?

Reply:
Maybe, this could be even generalized to identifiers containing "clk" or "clock". In that case, the pin name should preserved in the symbol to be able to distinguish the clocks.

Reply:
Addressed in f58ac3d1c334062ce6a5f18c47fcb13136f1a5bd

<img width="211" alt="image" src="https://github.com/user-attachments/assets/b196bdfb-d275-4c38-8da5-a3aafcb895c5" />

Reply:
@kjcolley7 : Thanks a lot, I like very much this enhancement!